actions/mongodb_explain_action.go

```diff
@@ -62,12 +67,12 @@ func (a *mongodbExplainAction) Run(ctx context.Context) ([]byte, error) {
 
 	err = bson.UnmarshalExtJSON([]byte(a.params.Query), true, &eq)
 	if err != nil {
-		return nil, fmt.Errorf("explain: unable to decode query %s: %s", a.params.Query, err)
+		return nil, errors.Wrapf(err, "Query: %s", a.params.Query)
 	}
 
 	res := client.Database("admin").RunCommand(ctx, eq.ExplainCmd())
 	if res.Err() != nil {
-		return nil, res.Err()
+		return nil, errCannotExplain
```

Comment:
I think we should at least log the original error (`err` and `res.Err()`) there and above